include/dali/core/mm/monotonic_resource.h

```diff
@@ -260,6 +272,10 @@ class monotonic_device_resource : public memory_resource {
   SmallVector<upstream_block, 8> blocks_;
 };
 
+using monotonic_host_resource = monotonic_memory_resource<memory_kind::host>;
```

Comment:
Maybe I'm missing something, but order doesn't have any default value nor is provided in this using statement.

Reply:
The default argument is defined in line 84.

Reply:
Missed that.